source/Octopus.Cli/Importers/ProjectImporter.cs

```diff
@@ -510,9 +510,17 @@ async Task<ProjectResource> ImportProject(ProjectResource project, string projec
             var existingProject = await Repository.Projects.FindByName(project.Name).ConfigureAwait(false);
             if (existingProject != null)
             {
+                if (existingProject.IsVersionControlled)
```

Comment:
`ProjectImporter` has been marked as deprecated since `2019`. I'll await a more ruthless change to remove this entirely until the integrations team can agree.